dist/index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-uWxY/CJNBR+1zjPWmfnSnVxwRheevXITnMqoEIeG1LJrdI0GlVs/9cVSyPYXdcSF" crossorigin="anonymous">
+    <title>Leaderboard</title>
+<script defer src="main.js"></script></head>
+<body>
+    <section class="main-section">
+        <h1 class="main-title">Leaderboard</h1>
+
+        <div class="main-div">
+
+            <div class="recent-scores">
+                
+                <div class="recent-scores--title">
+                    <h2>Recent scores</h2> 
+                    
+                    <button type="button" id="refresh-button" class="btn btn-success">Refresh</button>
+                </div>
+
+                <table class="table hide">
+                    <thead>
+                      <tr>
+                        <th class="thName hide" scope="col">Name</th>
+                        <th class="thScore hide" scope="col">Score</th>
+                      </tr>
+                    </thead>
+                    <tbody class="table-body">
+
+                    </tbody>
+                  </table>
+
+            </div>
+
+            <div class="add-scores">
+                <h2 class="add-scores-title">Add your score</h2>
+
+                <form class="form">
+                    <input 
+                    class="form-control"
+                    id="form-name" type="text" placeholder="Your name">
+
+                    <input 
+                    class="form-control"
+                    id="form-score" type="number" placeholder="Your score">
+
+                    <button id="submit-button" type="button" class="btn btn-warning">Submit</button>
+                </form>
```

Comment:
- [ ] OPTIONAL
Your _form_ is **accepting digits** as names. It will therefore be a nice implementation to add validations against your form inputs. Your form is accepting numbers as names

**Name appearing as digits:**
![image](https://user-images.githubusercontent.com/12745474/136485155-5eed9688-90f8-493b-beb9-5d2498a691b6.png)